lib/Gearman/Spawner/Server.pm

```diff
@@ -32,48 +33,45 @@ sub create {
 # start up a gearmand that exits when its parent process does. returns the
 # address of the listening server and its pid
 sub fork_gearmand {
-    my $class = shift;
-
-    # NB: this relies on Gearman::Spawner::Process allowing use of fork,
+	my $class = shift;
+    
+	# NB: this relies on Gearman::Spawner::Process allowing use of fork,
     # run_periodically, exit_with_parent, and loop as class methods instead of
     # object methods
 
-    # get an unused port from the OS
```

Comment:
That's a good suggestion and would be more robust fix. I have to think about that test case, probably needs some refactoring of the code or not if I found some new tricks for perl 🙂